### DISCUSSION TITLE
Vaultwarden server failed on arm64v8 platform

### DISCUSSION THREAD

**Discussion by PatrickHsu36** [dani-garcia/vaultwarden#3695] [Q&A]
Hello,
I deploy the arm64v8 docker container on a NAS for a long time, and the app works well till the recently version.
While I update to vaultwarden 1.28.1 and 1.29.0, the container can run successfully anymore.
The log of the container is as the followings.

**Error loading config:**
** SQLite database directory data does not exist or is not a directory**

I didn’t change any configuration to deploy the container. And the /data folder of Vaultwarden is already existed on the host from the 1st time I deploy the container till now.
The same Vaultwarden server 1.28.1 and 1.29.0 works well if using MariaDB or PostgreSQL database and on amd64 platform.

Could anyone help on this?
Thanks a lot!!

========

I rollback to use vaultwarden/server:1.27.0, and it works well, but from 1.28.0, it always failed with the error message.
So, the issue happened from 1.28.0.

**Comment by BlackDex** [comment]
As requested on discourse already, could you provide your setup on how you run your Vaultwarden, docker config like volumes env values etc...

**Reply by PatrickHsu36** [reply]
Hello,
(If I change the tag to 1.27.0) ,and then the container can be start correctly, and works as the before.
Change to 1.28.0 and then latest (1.29.0 now), the failed log will be there after start the container.

docker create -i -t --name=Vaultwarden \
	-p 32301:80 \
	-e ROCKET_TLS='{certs="/ssl/ssl.crt",key="/ssl/ssl.key"}' \
	-e ADMIN_TOKEN=$ADMINTOKEN \
	-e LOG_FILE='/data/bitwarden_rs.log' \
	-e LOG_LEVEL=warn \
	-v /usr/builtin/etc/certificate/:/ssl/:ro \
	-v /etc/localtime:/etc/localtime:ro \
	-v /share/Docker/Vaultwarden/bw-data:/data:rw \
	--restart unless-stopped \
	vaultwarden/server:latest

docker start Vaultwarden

Thanks a lot.
